tests/item_pickup_test.cpp

```diff
@@ -17,6 +18,14 @@ static bool container_has_item_type( const item &container, const itype_id it_ty
     } );
 }
 
+// Return true if avatar has an item with "uid" var set to the given string
+static bool avatar_has_item_with_uid( const avatar &they, const std::string uid_string )
+{
+    return they.has_item_with( [uid_string]( const item & cand ) {
+        return cand.get_var( "uid" ) == uid_string;
+    } );
+}
+
```

Comment:
If you don't have any objections I would request that we move this function to a helper class so my tests (which are testing autopickup functionality) can access it as well. Otherwise I have to duplicate it in my own test file.

Reply:
No problem. I refactored it a bit to make it more generic and moved it to `tests/player_helpers.cpp` - see 5caf4c3